### PR TITLE
netdev-rte-offloads: fix vxlan rte flows priority

### PR DESCRIPTION
**Pull request by evelineraine**
Signed-off-by: Eveline Raine <evelineraine@gmail.com>

### CODE REVIEW

**Comment by asafpenso** [comment]
merged

File: lib/netdev-rte-offloads.c [hunk 1]
```diff
@@ -1847,7 +1847,7 @@ netdev_vport_vxlan_add_rte_flow_offload(struct netdev_rte_port * rte_port,
 
     const struct rte_flow_attr flow_attr = {
         .group = rte_port->table_id,
-        .priority = 1,
+        .priority = 0,
```
Comment (OphirMunk) [inline lib/netdev-rte-offloads.c]: We should not set hard coded priorities in code. We should dynamically translate OVS flow priorities into DPDK priorities. We should also reserve an internal rte priority range for internal usage. For example the default TAG action should have a low priority and this priority should be hard coded (but probably lower than 1).

Comment (evelineraine) [reply lib/netdev-rte-offloads.c]: Do we have priorities for OVS dataplane flows? I haven't seen it in dpif/dump-flows.
Also, from DPDK docs, we get that not all hardware can support multiple priorities. I'm not sure how much hardware DPDK runs on supports our offloads, but not multiple priorities.

Comment (asafpenso) [reply lib/netdev-rte-offloads.c]: We'll look into that once we work on the upstream code.

**Pull request merged**